app/Http/ViewComposers/HeaderComposer.php

```diff
@@ -0,0 +1,32 @@
+<?php
+namespace App\Http\ViewComposers;
+
+use App\Country;
+use Illuminate\Contracts\View\View;
+use Illuminate\Support\Facades\Auth;
+
+
+class HeaderComposer {
+
+    public function compose(View $view) {
+
+        $user = Auth::user();
+        $default_country = Country::where('default', 1)->first();
```

Comment:
šitą reikėtų imti tik tuo atveju, jei useris neturi country. bus sutaupyta užklausa